### ISSUE TITLE
error in Xcode paytabs_ios.h not found

### ISSUE THREAD

**Issue by MuhammedFadel** [gardencoder/react-native-paytabs-library#9]
I'm using the newer version of this library as a submodule and I follow the manual instructions of the installing but I faced this error 

<img width="816" alt="63772371-a1d81300-c8e1-11e9-89c0-8fb221a3bb2f" src="https://user-images.githubusercontent.com/48424765/68742232-b4b8d480-0600-11ea-9429-d563c57e1a43.png">

thank you for creating this library,

**Comment by SABRYX** [comment]
I think you didn't add the paytabs-iOS.framework file to your project folder @MuhammedFadel

**Comment by MuhammedFadel** [comment]
> I think you didn't add the paytabs-iOS.framework file to your project folder @MuhammedFadel

thank you for responding..
Do mean in the main project folder same with previous library ?

**Comment by SABRYX** [comment]
> > I think you didn't add the paytabs-iOS.framework file to your project folder @MuhammedFadel
> 
> thank you for responding..
> Do mean in the main project folder same with the previous library ?

yes remove any files for older library and move them to trash then add the library to yourProject/yourProject folder

**Comment by MuhammedFadel** [comment]
> > > I think you didn't add the paytabs-iOS.framework file to your project folder @MuhammedFadel
> > 
> > 
> > thank you for responding..
> > Do mean in the main project folder same with the previous library ?
> 
> yes remove any files for older library and move them to trash then add the library to yourProject/yourProject folder

I will do thank you so much

**Comment by gardencoder** [comment]
@MuhammedFadel is it working fine?

**Comment by MuhammedFadel** [comment]
sorry for late responding..
no I still faced same issue also I get error with android after manual linking ( :@gardencoder/react-native-paytabs-library@1.0.3 not found ) 

I was using the old one but app crashed with some libraries like react-native-image-crop-picker and react-native-geolocation after remove it every thing works fine ( iOS / Android )

**Comment by MuhammedFadel** [comment]
> @MuhammedFadel is it working fine?

sorry for late responding..
no I still faced same issue also I get an error with Android after manual linking ( :@gardencoder/react-native-paytabs-library@1.0.3 not found ) 

I was using the old version but app crashed with some libraries like react-native-image-crop-picker and react-native-geolocation after remove it every thing works fine ( iOS / Android )
